modules/logging-create-clf.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/log-forwarding.adoc
+
+:_content-type: REFERENCE
+[id="logging-create-clf_{context}"]
+= Creating a log forwarder
+
+To create a log forwarder, you must create a `ClusterLogForwarder` CR that specifies the log input types that the service account can collect. You can also specify which outputs the logs can be forwarded to. If you are using multi log forwarder mode, you must also reference the service account in the `ClusterLogForwarder` CR.
+
+If you are using multi log forwarder mode on your cluster, you can create `ClusterLogForwarder` custom resources (CRs) in any namespace, using any name.
+If you are using legacy mode, the `ClusterLogForwarder` CR must be named `instance`, and must be created in the `openshift-logging` namespace.
+
+[IMPORTANT]
+====
+You need administrator permissions for the namespace where you create the `ClusterLogForwarder` CR.
+====
+
+.ClusterLogForwarder resource example
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: ClusterLogForwarder
+metadata:
+  name: <log_forwarder_name> <1>
+  namespace: <log_forwarder_namespace> <2>
+spec:
+  serviceAccount: <service_account_name> <3>
+  pipelines:
+   - inputRefs:
+     - <log_type> <4>
+     outputRefs:
+     - <output_type> <5>
+  outputs:
+  - name: <output_name> <6>
+    type: <output_type> <5>
+    url: <log_output_url> <7>
+# ...
+----
+<1> In legacy mode, the CR name must be `instance`. In multi log forwarder mode, you can use any name.
+<2> In legacy mode, the CR namespace must be `openshift-logging`. In multi log forwarder mode, you can use any namespace.
+<3> The name of your service account. The service account is only required in multi log forwarder mode.
+<4> The log types that are collected. The value for this field can be `audit` for audit logs, `application` for application logs, `infrastructure` for infrastructure logs, or a named input that has been defined for your application.
+<5> The type of output that you want to forward logs to. The value of this field can be `default`, `loki`, `kafka`, `elasticsearch`, `fluentdForward`, `syslog`, or `cloudwatch`.
```

Comment:
The supported output types should be updated per https://github.com/openshift/cluster-logging-operator/blob/master/bundle/manifests/logging.openshift.io_clusterlogforwarders.yaml#L379. Besides, if it's in mulitple CLF mode, `default` is not supported.

Reply:
@QiaolingTang per the earlier conversation with Jeff, we have additional Jiras to update the output docs. This PR is just moving them to their own section (the content is preexisting) so I would like to keep the scope to that for now, and we can update this section of docs later. See https://github.com/openshift/openshift-docs/pull/64935#discussion_r1337736020

> And if it's in mulitple CLF mode, default is not supported.

This part I think is within scope, so I'll update and add that, but updating the outputs more generally I think needs to be part of another Jira so that our scope doesn't get too large for this PR.